pkg/scheduler/apis/config/types.go

```diff
@@ -138,6 +138,9 @@ type KubeSchedulerProfile struct {
 // Enabled plugins are called in the order specified here, after default plugins. If they need to
 // be invoked before default plugins, default plugins must be disabled and re-enabled here in desired order.
 type Plugins struct {
+	// Enqueue is a list of plugins that should be invoked before adding pods to the scheduling queue.
+	Enqueue PluginSet
```

Comment:
I expected to see changes to pkg/scheduler/apis/config/validation/validation.go for this field, both that it is a valid pluginSet, and also that the field is empty when the new feature gate is turned off

Reply:
We need to add the new extension to https://github.com/kubernetes/kubernetes/blob/abcbaed0784baf5ed2382aae9705a8918f2daa18/pkg/scheduler/apis/config/validation/validation.go#L200, but it will be a no-op as of now. 

> and also that the field is empty when the new feature gate is turned off

The feature gate is for the schedulingGates feature, not the extension. I am not sure if we want to gate the extension itself.